child_compassion/models/compassion_hold.py

```diff
@@ -70,8 +70,14 @@ def check_hold_validity(self):
             ('expiration_date', '<',
              fields.Datetime.now())
         ])
-        for hold in expired_holds:
-            hold.active = False
+
+        for expired_hold in expired_holds:
+            child = self.env['compassion.child'].search([

```

Comment:
not useful to search, simply `child = expired_hold.child_id` or directly `expired_hold.child_id.active = False`